### DISCUSSION TITLE
(Slime) Ranchers

### DISCUSSION THREAD

**Discussion by VernaSaurus** [unitystation/unitystation#7896] [Ideas]
## What's a Rancher?
Well, simply put, Rancher is a job that takes the place of old xenobiology. You know, slimes and stuff. Alternatively it could be merged into Botany and be called "Farmer" or something.
### What do they do?
A rancher's job is to take care of their slimes. This is simple enough, as the various slimes just need some good-old TLC (and a bit of food). Slimes, upon being happy and healthy, will drop slime cores, without the need for killing your precious children (although that's still a valid option if you're heartless). The different slime cores do a wide variety of things, as well as being very lucrative if sold via cargo.
### What about old xenobiology?
Old xenobiology will be more focused around actual aliens, like Xenomorphs and other alien races.
### But why though.
Because it just felt weird that xenobiology was slimes, but I didn't feel like just getting rid of them would be very fun.

**Comment by fomalsd** [comment]
there are no slimes in yet, isn't this a bit premature?

**Comment by Call-Me-You** [comment]
> there are no slimes in yet, isn't this a bit premature?

He's into that kind of stuff, leave him alone.
in a corner
